tests/vectors/protocol-definitions/author-update.json

```
@@ -0,0 +1,27 @@
+{
+  "protocol": "http://recipient-update-protocol.xyz",
```

Super nit:
```suggestion
  "protocol": "http://author-update-protocol.xyz",
```
Also the protocol is a bit weird in that it only allows the owner to comment, but fair enough given this is all superficial anyways.

---

Updated string. You're right, it's not the most realistic example. This is mostly a test fixture than an insightful example to read.